octodns/provider/dyn.py

```diff
@@ -290,7 +290,8 @@ def traffic_directors(self):
             for td in get_all_dsf_services():
                 try:
                     fqdn, _type = td.label.split(':', 1)
-                except ValueError:
+                except ValueError as e:
+                    self.log.warn("Failed to load TraficDirector '{}': {}".format(td.label, e))
```

Comment:
The logging functions can actually do their own formatting, so it's slightly preferable to do 

```
self.log.warn("Failed to load TraficDirector '%s': %s", td.label, e)
```

I realize there's a bit of inconsistency in octoDNS with this currently, but it's something I plan to address so would like to make sure that new additions go with the "preferred" route.